Makefile

```diff
@@ -47,6 +47,11 @@ BUILD_DEPS+=ffi-version-check
 
 .PHONY: ffi-version-check
 
+build/rice-box.go:
+	go run github.com/GeertJohan/go.rice/rice embed-go -i ./build
+
+BUILD_DEPS+=build/rice-box.go
```

Comment:
This will generate a massive Go file which slows down builds, and likely IDEs.

At this point we should probably just switch builtin Go embed (https://golang.org/pkg/embed/)

We only use rice in `builtin/` in a few places:
```
build/bootstrap.go
18:     b := rice.MustFindBox("bootstrap")

build/openrpc.go
31:     data := rice.MustFindBox("openrpc").MustBytes("full.json.gz")
36:     data := rice.MustFindBox("openrpc").MustBytes("miner.json.gz")
41:     data := rice.MustFindBox("openrpc").MustBytes("worker.json.gz")

build/parameters.go
6:      return rice.MustFindBox("proof-params").MustBytes("parameters.json")
```

Reply:
Good point about the huge source file. 

I have swapped out go rice for go:embed for everything under build. There is still a rice box in the townhall and fountain and that can be removed as well if we want to switch all embeded files to go:embed.


This feature is only available on go 1.16+ a